stories/navigation/Tabs.stories.js

```diff
@@ -14,8 +14,8 @@ const tabs = [
     onClick: action('Details Clicked'),
   },
   {
-    content: 'Keys',
-    onClick: action('Keys Clicked'),
+    content: 'More Details',
+    onClick: action('More Details Clicked'),
```

Comment:
Would it be possible here to mock out an example of how it functions when a new tab is selected?

Reply:
There is a very short example in the stories. But isn't that great. We should probably add a `useTabs` hook later to handle this state
```js
const Example = () => {
  const [i, seti] = React.useState(0);
  return <Tabs selected={i} onSelect={ind => seti(ind)} tabs={tabs} />;
};
```